packages/api/src/endpoints/chat/HelixChatApi.ts

```diff
@@ -251,6 +256,35 @@ export class HelixChatApi extends BaseApi {
 		return new HelixPrivilegedChatSettings(result.data[0]);
 	}
 
+	/**
+	 * Sends a chat message to a broadcaster's chat.
+	 *
+	 * This uses the token of the broadcaster by default.
+	 * If you want to execute this in the context of another user (who has to be moderator of the channel)
```

Comment:
The user does not need to be a moderator.

Additionally, there should probably be a way to send explicitly via the app token capabilities - this probably needs to be a separate method, as the sender/scope logic doesn't work in a `.withoutUser` context.

Reply:
@d-fischer I've updated the JSDoc. I don't know how you want to implement the separate app token method, so you'd either need to provide an example of an existing implementation of this, or add it in yourself. You have write access to my branch for convenience.